vcr/credential/validator.go

```diff
@@ -302,6 +305,16 @@ func (d x509CredentialValidator) Validate(credential vc.VerifiableCredential) er
 		cert, _ := x509.ParseCertificate(der)
 		chain[i] = cert
 	}
+
+	// Check credential's time validity v.s. certificate's time validity
+	leafCert := chain[0]
+	if credential.ExpirationDate != nil && credential.ExpirationDate.After(leafCert.NotAfter) {
```

Comment:
is it required to set an `credential.ExpirationDate` since the cert is guaranteed to have an expiration date?